docs/docs/guides/migrate/sources/keycloak.md

```diff
@@ -0,0 +1,455 @@
+
+On the menu on the left, select **Clients** and click the button **Create client**. 
+
+<img src="/docs/img/guides/migrate/keycloak-06.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+Leave the client type as **OpenID Connect**, fill in the desired **Client ID** and **Name**, then click on **Next**:
+
+<img src="/docs/img/guides/migrate/keycloak-07.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+You don't need to make any changes to the **Capability config**. Go ahead and click **Next**.
+
+<img src="/docs/img/guides/migrate/keycloak-08.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+The web application used for this demo will run on `http://localhost:4200/`. In order to allow login and logout from the application, this client needs to be configured to accept your application URL in **Login settings**. Edit **Root URL**, **Valid redirect URI**, and **Valid post logout redirect URIs** to point to your application URLs. Without this configuration, Keycloak will refuse login and logout from your application due to security concerns.
+ 
+Additionally, **Web origins** needs to be configured to support required cross-domain requests; otherwise, the request will be blocked on all browsers due to security concerns. To make this application work, fill in all the fields as shown below. 
+
+<img src="/docs/img/guides/migrate/keycloak-09.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+Finally, create the client by clicking **Save**.
+
+<img src="/docs/img/guides/migrate/keycloak-10.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+### Create user in Keycloak
+
+The last thing you need to do in Keycloak is to create at least one new user. This user will be able to log into your application.
+
+On the menu on the left, select **Users**, and click **Add user**. Fill in the username, email, and first and last names, and mark the email as verified. Click on **Create** to create a new user:
+
+<img src="/docs/img/guides/migrate/keycloak-11.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+<img src="/docs/img/guides/migrate/keycloak-12.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+<img src="/docs/img/guides/migrate/keycloak-13.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+Now you should attach a password to this user. Select the **Credentials** tab and click **Set password**.
+
+<img src="/docs/img/guides/migrate/keycloak-14.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+ On the new modal panel, input the desired password and select **Save**.
+
+<img src="/docs/img/guides/migrate/keycloak-15.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+<img src="/docs/img/guides/migrate/keycloak-16.png" alt="Migrating users from Keycloak to ZITADEL"/>
```

Comment:
The target audience is moving away from keycloak. Presumably they do not need a getting stated guide on starting it, right?

Reply:
Yes, can do. Left it as it is because it was a part of the original article.